hackathon_site/dashboard/frontend/src/assets/images/icons/redtype.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It would get bloated to upload a bunch of circle SVGs for each category color, you can use some circle-looking icon from MUI and set the color with css based off your `type` prop. Here's one that might work:
```javascript
import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
```

You can search through all the icons here: https://material-ui.com/components/material-icons/